estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,50 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer"
+    _order = "price desc"
+    _check_price = models.Constraint("CHECK(price>0)", "Le prix doit être strictement positif.")
+
+    price = fields.Float()
+    status = fields.Selection(copy=False, selection=[('Accepted', 'Accepted'), ('Refused', 'Refused')])
+    partner_id = fields.Many2one("res.partner", required=True)
+    property_id = fields.Many2one("estate.property", required=True)
+    property_type_id = fields.Many2one(related="property_id.property_type_id", store=True)
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(compute="_compute_deadline", inverse="_inverse_deadline")
+
+    @api.depends("validity")
+    def _compute_deadline(self):
+        for record in self:
+            record.date_deadline = fields.Date.add(record.create_date or fields.Date.today(), days=record.validity)
+
+    def _inverse_deadline(self):
+        for record in self:
+            record.validity = (record.date_deadline - fields.Date.to_date(record.create_date)).days
+
+    def action_accept(self):
+        self.status = "Accepted"
```

Comment:
You can protect the db and raise an exception if self contains several records.

```suggestion
    def action_accept(self):
        self.ensure_one()
        self.status = "Accepted"
```